plugin.rb

```diff
@@ -279,6 +279,23 @@ def self.github_organizations
     end
   end
 
+  add_to_class(:user, :can_review?) do
+    @can_review ||= begin
```

Comment:
It seems like this method does not consider `SiteSetting.code_review_enabled` at the moment. Should it return false if the plugin is disabled?

Reply:
`add_to_class` automatically checks that.

Reply:
Thanks! I didn't know that. I have been manually checking site settings since forever :sweat_smile: